content/administration/on_premise/packages.rst

```diff
@@ -100,9 +100,9 @@ be downloaded from the `Odoo download page <https://www.odoo.com/page/download>`
    .. group-tab:: Debian/Ubuntu
 
       .. note::
-         Odoo {CURRENT_MAJOR_VERSION} 'deb' package currently supports `Debian Buster
-         <https://www.debian.org/releases/buster/>`_ and `Ubuntu 18.04
-         <https://releases.ubuntu.com/18.04>`_ or above.
+         Odoo {CURRENT_MAJOR_VERSION} 'deb' package currently supports `Debian Bookworm (12)
+         <https://www.debian.org/releases/bookworm/>`_ and `Ubuntu Jammy (22.04LTS)
```

Comment:
```suggestion
         <https://www.debian.org/releases/bookworm/>`_ and `Ubuntu Jammy (22.04 LTS)
```

Reply:
Ubuntu specifically is spelling it `22.04LTS`, I don't like it either, but I'm going to let it this way

Reply:
Not on their [website](https://releases.ubuntu.com/) at least :sweat_smile: 
But it doesn't matter much anyway :)